spec/visual/samples/ada

```diff
@@ -0,0 +1,343 @@
+------------------------------------------------------------------------------
```

Comment:
I'm not sure it's appropriate to use a GPL'd file as a sample for Rouge. Rouge is MIT licensed, and while it may not be a problem to include it as a sample, it might be preferable to use another sample. Also, the point of the sample file is to show the effect of the different rules of the lexer. Since this lexer is very basic, I don't think it's necessary to have such a large sample.